tests/test_astronomer_commander.py

```diff
@@ -0,0 +1,61 @@
+from tests.helm_template_generator import render_chart
```

Comment:
These work locally at least. Not sure if we want to get them into the ci conf

Reply:
I've actually [started work on backporting pytest to 0.23](https://github.com/astronomer/astronomer/commit/3cd724f8afcd80ccbe0aab4daca5bb7ad6646d4c). Let's worry about that in the other branch, and just accept this as groundwork for now, while not going as far as putting it in CI. If we do any more work on 0.23 we'll push for full pytest support.